src/sankey/index.js

```diff
@@ -0,0 +1,137 @@
+import React, {PropTypes, Component} from 'react';
+import {voronoi} from 'd3-voronoi';
+import {sankey} from 'd3-sankey-align';
+
+import {DISCRETE_COLOR_RANGE} from 'theme';
+
+const noop = f => f;
```

Comment:
i would hardly say this is a variable. also that's definitely not true! look immediately above there are a bunch of screaming snake case constants.